ci-tools/host-runner/README.md

```diff
@@ -0,0 +1,255 @@
+# Introduction
+
+## NixOS system
+
+This is a NixOS installation for Caliptra host runners. A host runnner is a Raspberry PI that oversees a cluster of FPGAs running in the Caliptra CI.
+
+## Setting up a NixOS system for cross-compilation
```

Comment:
I'm a bit unclear on what this means. Is the to build NixOS images on a development machine? Probably worth a top level sentence on what the goal is.

Reply:
Re-phrased this a bit to try and clear things up.